satellite/audit/queue_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/pkg/storj"
+)
+
+func TestQueue(t *testing.T) {
+	queue := &Queue{
+		Queue: []storj.Path{},
+	}
+	_, err := queue.Next()
+	require.True(t, ErrEmptyQueue.Has(err))
```

Comment:
I wouldn't bother with these since it's often more informative to look at the code. There are cases where this extra tag is useful, e.g. loops, where you cannot easily deduce which test is failing.